docs/pages/repo/docs/core-concepts/monorepos/ignoring.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Ignoring tasks
+description: Never do the same work twice.
+---
+
+# Ignoring Tasks
+
+[Build caching](/repo/docs/core-concepts/caching) will dramatically speed up your build pipelines - but you can do even better by using `npx turbo-ignore`. If your workspace is unaffected by your code changes, you can completely ignore executing a task altogether.
+
+## Using `turbo-ignore`
+
+To skip unaffected work with `npx turbo-ignore`, all you'll need is to make sure your git history is available.
+
+By default, `turbo-ignore` will run the build task in your current working directory. Under the hood, `turbo-ignore` compares the output of `--dry=json` from your previous commit to your current commit.
+
+Here's an example of the command that it will build:
+
+```bash
+npx turbo run build --filter=@example/web...[3c8387ffd98b751305fe3f0284befdd00cbd4610] --dry=json
+```
+
+Note that this command does not _execute_ the build task. Instead, it checks if a **dry run** of a build is different from the **dry run** of the task from the previous commit.
+
+## Customizing behavior
+
+If you'd like to change the default behavior, there are a few arguments available.
+
+- `--workspace`: Runs `turbo-ignore` in the workspace with the name provided
+- `--task`: Runs `turbo-ignore` with the specified task
+- `--directory`: Specify a directory to run `turbo-ignore` in
+- `--fallback`: A ref/head to compare against if no previously deployed SHA is available
```

Comment:
"previously deployed SHA" is a bit confusing here. Are we talking specifically about Vercel? Are we talking specifically of `build` tasks (assuming `build` tasks mean what we think it means for everyone)

Reply:
@tknickman I lack context on this one. Might I be able to suggest a suggestion from ya? 😬

Reply:
Yep, previously deployed SHA is Vercel only

> The git SHA of the last successful deployment for the project and branch. Example: fa1eade47b73733d6312d5abfad33ce9e4068080. NOTE: This Variable is only exposed when an Ignored Build Step is provided.

https://vercel.com/docs/concepts/projects/environment-variables#system-environment-variables